File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/expand/ArtistViewHolder.java
package com.thoughtbot.expandablerecyclerview.sample.expand;

import android.view.View;
import android.widget.TextView;
import com.thoughtbot.expandablerecyclerview.sample.R;
import com.thoughtbot.expandablerecyclerview.viewholders.ChildViewHolder;

public class ArtistViewHolder extends ChildViewHolder {

  private TextView childTextView;

  public ArtistViewHolder(View itemView) {
    super(itemView);
    childTextView = (TextView) itemView.findViewById(R.id.list_item_artist_name);
  }

  public void setArtistName(String name) {
    childTextView.setText(name);
  }
}

File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/expand/ArtistViewHolder.kt
package com.thoughtbot.expandablerecyclerview.sample.expand

import android.view.View
import android.widget.TextView
import com.thoughtbot.expandablerecyclerview.sample.R
import com.thoughtbot.expandablerecyclerview.viewholders.ChildViewHolder

class ArtistViewHolder(itemView: View) : ChildViewHolder(itemView) {
    private val childTextView: TextView
    fun setArtistName(name: String?) {
        childTextView.text = name
    }

    init {
        childTextView = itemView.findViewById<View>(R.id.list_item_artist_name) as TextView
    }
}